rest_framework/fields.py

```diff
@@ -178,7 +204,7 @@ def __init__(self, read_only=False, write_only=False,
         self.required = required
         self.default = default
         self.source = source
-        self.initial = self.initial if (initial is empty) else initial
+        self.initial = initial
```

Comment:
For example, when this PR includes changes like this line it's not immediately clear if this is a behavioral change or not. Addressing individual arguments or sets of arguments per pull request would help make that clearer.